FCCSW_ecal/run_thetamodulemerged.py

```diff
@@ -282,11 +282,12 @@
     saveHCalTool.CaloHits.Path = hcalBarrelHitsName
```

Comment:
I would suggest to rename saveHCalTool to saveHCalBarrelTool now that we have also the tool for the endcap ;)

Reply:
done